integration_test/helpers/helpers.dart

```diff
@@ -117,7 +117,9 @@ Future<void> issueCredentials(
     expect(attributeTexts[i * 2 + 1], attributeEntries[i].value);
   }
 
-  var acceptButtonFinder = find.byKey(const Key('bottom_bar_primary'));
+  final acceptButtonFinder = find.byKey(
```

Comment:
One very small nitpicky thingy: when declineOffer=true the button found is not an 'acceptButton', so you might want to consider renaming this to something like actionButtonFinder or buttonFinder